### PR TITLE
set the right zoomlevel for lk25 and lk50 backlinks from zeitreise la…

### PR DESCRIPTION
**Pull request by ltclm**
…yer tooltip

proposed solution for https://github.com/geoadmin/mf-chsdi3/issues/2142

[Testlink ](https://mf-geoadmin3.dev.bgdi.ch/?topic=ech&lang=de&bgLayer=ch.swisstopo.swissimage&api_url=dev_ltclm_lk25_lk50_fullres&X=128527.50&Y=630625.00&zoom=8&layers=ch.swisstopo.zeitreihen&layers_timestamp=19951231&time=1995)

### CODE REVIEW

File: chsdi/templates/historicalmaps/viewer.mako [hunk 1]
```diff
@@ -18,6 +18,13 @@
   title += ': ' + pageTitle
   loaderUrl = h.make_agnostic(route_url('ga_api', request))
 
+  if c.get('title') == 'kartenwerk_lk25':
```
Comment (gjn) [inline chsdi/templates/historicalmaps/viewer.mako]: Can't we used layerBodId here? I think it would be more stable as the title.

Comment (ltclm) [inline chsdi/templates/historicalmaps/viewer.mako]: layerbodid is always ``ch.swisstopo.zeitreihen``

Comment (gjn) [inline chsdi/templates/historicalmaps/viewer.mako]: Too bad.

**Comment by gjn** [comment]
This branch has a conflict. Please resolve and merge once jenkins is happy.

**Comment by ltclm** [comment]
thats the jenkins error:
```
======================================================================
FAIL: Failure: AssertionError (ch.blw.klimaeignung-koernermais)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/var/lib/jenkins/jobs/PR_chsdi3/workspace/.venv/local/lib/python2.7/site-packages/nose-1.3.7-py2.7.egg/nose/loader.py", line 251, in generate
    for test in g():
  File "/var/lib/jenkins/jobs/PR_chsdi3/workspace/chsdi/tests/integration/test_layers.py", line 144, in test_all_htmlpopups
    for layer, feature, extended in lc.ilayersWithFeatures():
  File "/var/lib/jenkins/jobs/PR_chsdi3/workspace/chsdi/tests/integration/test_layers.py", line 82, in ilayersWithFeatures
    assert (models is not None and len(models) > 0), layer
AssertionError: ch.blw.klimaeignung-koernermais

======================================================================
FAIL: Failure: AssertionError (ch.blw.klimaeignung-koernermais)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/var/lib/jenkins/jobs/PR_chsdi3/workspace/.venv/local/lib/python2.7/site-packages/nose-1.3.7-py2.7.egg/nose/loader.py", line 251, in generate
    for test in g():
  File "/var/lib/jenkins/jobs/PR_chsdi3/workspace/chsdi/tests/integration/test_layers.py", line 156, in test_all_ids_mapping
    for layerId, featureId, model, primaryKeyColumn in lc.ilayersAllModels():
  File "/var/lib/jenkins/jobs/PR_chsdi3/workspace/chsdi/tests/integration/test_layers.py", line 66, in ilayersAllModels
    assert (models is not None and len(models) > 0), layer
AssertionError: ch.blw.klimaeignung-koernermais
```
any idea where this problem could come from...

**Comment by gjn** [comment]
We have the same on production. Didn't have time yet to analyse. But it's related to the update we did yesterday. Seems like there's no model for this layer? ping @alcapat ?

Anyhow, I'll merge this PR.

**Pull request merged**